web/app.php

```diff
@@ -5,7 +5,7 @@
 use Symfony\Component\ClassLoader\ApcClassLoader;
 use Symfony\Component\HttpFoundation\Request;
 
-/**
+/*
```

Comment:
this is wrong. The right fix would be to fix the content to say ``@var $loader ...``